src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -2051,11 +2052,13 @@ namespace cryptonote
       unsigned int b = 0;
       const time_t time_boundary = now - static_cast<time_t>(seconds[n]);
       for (time_t ts: timestamps) b += ts >= time_boundary;
-      const double p = probability(b, seconds[n] / DIFFICULTY_TARGET_V2);
-      MDEBUG("blocks in the last " << seconds[n] / 60 << " minutes: " << b << " (probability " << p << ")");
+      const double dur = seconds[n] / DIFFICULTY_TARGET_V2; // duration of block generation we are measuring
```

Comment:
The computation here is fine, but the variable naming and comment seem confusing: The unit of the computed value is blocks (`seconds / (seconds / block)`), so this is the expected/mean/target block count for the given period length; it's not a duration of any kind.

Reply:
Thanks. I updated the names and comments.